### PR TITLE
Portfolio project: Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by hamzaw9**
This project aims to create an appealing portfolio to showcase my projects/skills. 
But for now, this project contains only 2 sections:

- Header Section (Logo & navbar)

- Main Section (Main text & social media links)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Start: Header -->
+    <div class="wrapper">
+      <header class="header">
+        <a href="#">My Logo</a>
+        <i class="fa-solid fa-bars"></i>
```
Comment (hiromataba) [inline index.html]: - [ ] The hamburger button should have  a nav tag, the nav tag is for semantics reason, and the right position for convention and user experience for mobile devices

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <h4>LET’S CONNECT</h4>
+        <section class="social-media">
+          <a href="#"><img src="images/twitter icon.svg" alt="Twitter icon"></a>
+          <a href="#"><img src="images/linkedin icon.svg" alt="Linkedin icon"></a>
+          <a href="#"><img src="images/m icon.svg" alt="M icon"></a>
+          <a href="#"><img src="images/github icon.svg" alt="Github icon"></a>
+          <a href="#"><img src="images/peace icon.svg" alt="Peace icon"></a>
+        </section>
+      </main>
```
Comment (hiromataba) [inline index.html]: - You should use the `section` tag to wrap this section actually , It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <div class="wrapper">
+      <header class="header">
+        <a href="#">My Logo</a>
+        <i class="fa-solid fa-bars"></i>
+      </header>
+     <!-- End: Header -->
+
+     <!-- Start: Main-->
+      <main>
+        <h2>I’m Hamza Baig. Glad to see <br> you!</h2>
```
Comment (hiromataba) [inline index.html]: - [ ] Your main title should be a h1 tag, If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

File: index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <h4>LET’S CONNECT</h4>
+        <section class="social-media">
+          <a href="#"><img src="images/twitter icon.svg" alt="Twitter icon"></a>
+          <a href="#"><img src="images/linkedin icon.svg" alt="Linkedin icon"></a>
+          <a href="#"><img src="images/m icon.svg" alt="M icon"></a>
+          <a href="#"><img src="images/github icon.svg" alt="Github icon"></a>
+          <a href="#"><img src="images/peace icon.svg" alt="Peace icon"></a>
```
Comment (hiromataba) [inline index.html]: - [ ]  The list of icons should  use a `ul` tag to group them, and `li` per icon; each icon has `a` link (a tag).
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @hamzaw9   :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hamzaw9** [comment]
**Incorporate reviewer feedback: update index.html and style.css files to improve accessibility and layout.**

File: index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <h4>LET’S CONNECT</h4>
+        <ul class="social-media">
+          <li><a href="#"><img src="images/twitter icon.svg" alt="Twitter icon"></a></li>
+          <li><a href="#"><img src="images/linkedin icon.svg" alt="Linkedin icon"></a></li>
+          <li><a href="#"><img src="images/m icon.svg" alt="M icon"></a></li>
+          <li><a href="#"><img src="images/github icon.svg" alt="Github icon"></a></li>
+          <li><a href="#"><img src="images/peace icon.svg" alt="Peace icon"></a></li>
+        </ul>
+      </section>
```
Comment (taldr27) [inline index.html:32]: Great work incorporating the suggestions from the previous reviewer! I do **have a few optional suggestions that could make your project look even better and more closely match** the [Figma template provided](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=NcRw8dFhHkmW6sux-0):

- [ ] **[OPTIONAL]** Would you mind checking the `padding` you added to your `wrapper`? You could remove it from there and instead apply `padding` **for each section separately**. For example, you could add some `padding` for the `header` and some `padding` for your section. This small adjustment will help your **background image look more similar to the template**. 😉

  ### Your design:
  image

  ### Figma design:
  image

- [ ] **[OPTIONAL]** Additionally, you could try adding `align-items: center` to center the **Medium** icon. This **will help make the design more visually appealing and professional**. 🎯

  image

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hi @hamzaw9, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉 🎯💯🙌👍🤓
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.
✅ Pull request has a descriptive title and summary.

## Optional suggestions 💭

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**